src/pyprojroot/__init__.py

```diff
@@ -1,3 +1,12 @@
-from .criterion import *
+from .criterion import as_root_criterion, has_dir, has_file
 from .root import find_root, find_root_with_reason
 from .here import here
+
+__all__ = [
```

Comment:
I don't think you need this since everything you want to export is explicitly imported.